### ISSUE TITLE
In IMDB dataset , Number of training examples: 0 [ solved by myself]

### ISSUE THREAD

**Issue by sc1884** [pytorch/text#667]
## ❓ Questions and Help

```
from torchtext import datasets

train_data, test_data = datasets.IMDB.splits(TEXT, LABEL,root='.data')
datasets.IMDB.splits()

print(f'Number of training examples: {len(train_data)}')
print(f'Number of testing examples: {len(test_data)}')
```
output 
```
Number of training examples: 0
Number of testing examples: 0
```
There are "./.data/imdb" file in my current folder downloaded by  ''datasets.IMDB.splits()''. 

The problem is the output should be : 
```
Number of training examples: 25000
Number of testing examples: 25000
```
from  https://github.com/bentrevett/pytorch-sentiment-analysis/blob/master/1%20-%20Simple%20Sentiment%20Analysis.ipynb

<------------------------------------------------------->
😂, After I typed these word above just now.
I return back to delete all the ".data" folder and rerun `train_data, test_data = datasets.IMDB.splits(TEXT, LABEL,root='.data')`. All results seem to be right.
I guess the reason is that I force-stop the jupyter notebook when torchtext download IMDB datasets.
I do this because I want to use  proxy, and then I run the code in shell. Maybe my operations results some error.

**Comment by zhangguanheng66** [comment]
You should wait until the full zip/tar files are downloaded.
